projects/sport-data-valley/seeds/questionnaires/mmm_questionnaire.rb

```diff
@@ -0,0 +1,508 @@
+# frozen_string_literal: true
+
+MMM_QUESTIONNAIRE_NAME = 'move_mood_motivation'
+
+db_name = MMM_QUESTIONNAIRE_NAME
+questionnaire = Questionnaire.find_by(name: db_name)
+questionnaire ||= Questionnaire.new(name: db_name)
+questionnaire.key = db_name
+
```

Comment:
There is the convention that the key of the questionnaire should be the same as the file name of the questionnaire (minus the .rb). That way, if you know the key of a questionnaire, it's always easy to look up the file for it.

For example, if you look at the first few lines of `braf_rheumatism.rb`, it says:

```ruby
db_title = ''
db_name1 = 'braf_rheumatism'
dagboek1 = Questionnaire.find_by(key: File.basename(__FILE__)[0...-3])
dagboek1 ||= Questionnaire.new(key: File.basename(__FILE__)[0...-3])
dagboek1.name = db_name1
```

Here it automatically finds the questionnaire by the `key`, which is the basename of the file (minus the `.rb` extension). Also here you can see that I set the name to the `braf_rheumatism`, which is the same as the `key` which is the same as the base file name.

Btw the reason that the title of a questionnaire empty because the `questionnaire.title` attribute does not support localization. So if we have a localized questionnaire, and a title in Dutch, the Dutch title will always show up even if a person is filling out the questionnaire in English. That's why we usually have a `raw` block at the start that has some content to show a "title" of a questionnaire. Since this content supports localization, it works.

Reply:
Ok, will change then :smile: